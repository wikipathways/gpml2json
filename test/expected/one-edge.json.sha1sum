80abd4e7c5917b84ff1224b7906d3207a95af28c ?-
