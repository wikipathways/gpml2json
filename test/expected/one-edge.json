{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["pvjsgeneratedida01"],"drawAs":"rect","gpmlElementName":"Pathway","height":237,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":197,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20140130","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/3287824f5b1532096f1a3bfadaaf3ec38debf491d032f36674ec63fe3ad077f3"},"entityMap":{"pvjsgeneratedida01":{"borderWidth":1,"zIndex":12288,"points":[{"x":96,"y":127},{"x":167,"y":207}],"color":"#000000","drawAs":"StraightLine","id":"pvjsgeneratedida01","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction"}}}
